lib/ansible/modules/cloud/amazon/sqs_queue.py

```diff
@@ -23,48 +23,86 @@
   - Alan Loi (@loia)
   - Fernando Jose Pando (@nand0p)
   - Nadir Lloret (@nadirollo)
+  - Dennis Podkovyrin (@sbj-ss)
 requirements:
-  - "boto >= 2.33.0"
+  - boto3
 options:
   state:
     description:
       - Create or delete the queue
-    required: false
     choices: ['present', 'absent']
     default: 'present'
   name:
     description:
       - Name of the queue.
     required: true
-  default_visibility_timeout:
+  queue_type:
+    description:
+      - Standard or FIFO queue.
```

Comment:
This should probably mention that the queue_type can only be selected at creation time and is *ignored* when making other changes